net/natmap/patches/100-don-t-get-commit-id-from-git.patch

```diff
@@ -0,0 +1,30 @@
+From ac0492d4d480948a3cd2790de3f9eeaccb0527ff Mon Sep 17 00:00:00 2001
```

Comment:
run `make package/natmap/refresh` to refresh your patch.

Reply:
Thanks.